utils/luci-app-owm/files/owm.lua

```diff
@@ -6,6 +6,34 @@ require("luci.sys")
 require("nixio.fs")
 require("luci.httpclient")
 
+-- Print help text
+function print_help()
+	print("owm.lua - Tool for registering routers at openwifimap.net\n")
+	print("Options:\
+        -\-help|-h:\tprint this text\
+	\
+        -\-dry-run:\tcheck if owm.lua is working (does not paste any data).\
+	\t\tWith this option you can check for errors in your\
+	\t\tconfiguration and test the transmission of data to\
+	\t\tthe map.")
+	print("\nIf invoked without any options, this tool will try to register your node at\
+the community-map, but will execute silently. To work correctly, this tool will need\
+at least the geo-location of the node (check with -\-dry-run).\
+\
+To send the data to another server, you can give its URL in /etc/config/freifunk (option: owm_api).\
```

Comment:
Maybe change the text to "To override the server used by this script, set freifunk.community.owm_api" or something similar.  That way, the config filename and the section are clear and can also be easily copied into a call to "uci show" or "uci set".

Reply:
thank you. That's my aim. :) I changed the sentence as you requested.

Reply:
Yeah!